fdbserver/fdbserver.actor.cpp

```diff
@@ -2028,6 +2028,8 @@ int main(int argc, char* argv[]) {
 			enableGeneralBuggify();
 		} else {
 			disableGeneralBuggify();
+			// When buggify is disabled, also disable global fault injection
+			opts.faultInjectionEnabled = false;
```

Comment:
Why do we want to disable the fault injection? I think the buggify is not exactly same as the fault injection. Please correct me if I'm wrong. Thanks!